sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -764,35 +894,119 @@ public Write withoutValidation() {
     }
 
     /**
-     * Returns a new {@link BigtableIO.Write} that will write using the given Cloud Bigtable service
-     * implementation.
+     * Returns a new {@link BigtableIO.Write} that will use an official Bigtable emulator.
      *
      * <p>This is used for testing.
+     */
+    @VisibleForTesting
```

Comment:
I think this should be available for end users to test their pipelines using direct runner and the emulator

Reply:
There wasn't a way for end users to set BigtableService before. So I think we should keep the behaviors the same for now? We can always expose it later?

Reply:
ok